viscy/scripts/network_diagram.py

```diff
@@ -5,15 +5,15 @@
 
 # %%
 model = VSUNet(
```

Comment:
thanks for this script @ziw-liu. This is very helpful to just explore specs and diagrams of different architectures.

Reply:
The svg outputs from torchview look great!